### PR TITLE
Remove space between cast and variable in several places

### PR DESCRIPTION
**Pull request by necropotame**
And also:
* fix variable names in several places
* fix spaces around some pointers in several places

### CODE REVIEW

**Comment by heinrich5991** [comment]
CI says that there are build errors. Otherwise, looks OK to me. Anyone against a large style fix?

**Comment by heinrich5991** [comment]
And the first line of the commit message shouldn't be this long. ^^
```
Fix some style issues

<detailed explanation>
```

File: src/base/color.h [hunk 1]
```diff
@@ -25,24 +25,24 @@ inline float HueToRgb(float v1, float v2, float h)
 
 inline float RgbToHue(vec3 rgb)
```
Comment (Learath2) [inline src/base/color.h]: Don't think we should change styling in `src/base/*` at all.

**Comment by Learath2** [comment]
Other than that should merge.

**Comment by heinrich5991** [comment]
Other than that and the compile error.

**Comment by necropotame** [comment]
I've reverted color.h (I don't want to change ./src/base me too, but I've didn't pay attention that it was in this directory when I used grep ^^)
The compilation error is fixed too (Ive forgotten that I should use "make everything")

**Comment by heinrich5991** [comment]
Thanks.

**Pull request merged**